estate/models/estate_property.py

```diff
@@ -51,5 +51,30 @@ class EstateProperty(models.Model):
     tag_ids = fields.Many2many("property.tag")
     partner_id = fields.Many2one("res.partner")
     offer_ids = fields.One2many("property.offer", "property_id")
+    total_area = fields.Float(compute="_compute_total_area", string="Total Area (sqm)")
+    best_offer = fields.Float(compute="_compute_best_price", string="Best Offer")
```

Comment:
I think that the string is exactly the same as the name of the field

```suggestion
    best_offer = fields.Float(compute="_compute_best_price")
```